README.md

```diff
@@ -29,12 +29,13 @@ metadata:
 ## Prerequisites
 - go
 - git
-- kustomize.io
 - sed
 - jq
 - yq (python-yq from https://github.com/kislyuk/yq#installation, other distributions may not work)
 - docker
 
+Note: kustomize `v4.0.5` is required for most tasks. It is downloaded automatically to the `.kustomize` folder in this repo when required. This downloaded version is used regardless of whether or not kustomize is already installed on the system.
```

Comment:
Not really for most but it's not really matter.

Reply:
By number, maybe not, but it's needed for `make install` and `make uninstall` :)